core/arch/arm/plat-imx/conf.mk

```diff
@@ -244,3 +247,18 @@ CFG_SHMEM_SIZE ?= 0x00200000
 CFG_CRYPTO_SIZE_OPTIMIZATION ?= n
 CFG_WITH_STACK_CANARIES ?= y
 CFG_MMAP_REGIONS ?= 24
+
+# Almost all platforms include CAAM HW Modules, except the
+# one force to be disabled
+CFG_NXP_CAAM ?= y
```

Comment:
Will this break the image booting using vendor/upstream uboot + vendor/upstream linux? Or please share some public doc somewhere to use the image built out with this "y"

Reply:
removed